python-client/examples/dag/worker.py

```diff
@@ -0,0 +1,40 @@
+from hatchet import workflow, step, Worker, Context
+
+@workflow(on_events=["user:create"])
```

Comment:
nit style choice: most of the client libs I've used in Python will instantiate (or import) an object with config and then call decorators on that object.

FastAPI/Flask/Celery all use a similar pattern. I've had some instances where I've needed multiple Celery instances in one app because of bad decisions.

```py
hatchet = Hatchet(config)

@hatchet.workflow(...)
class MyWorkflow:

...

    @hatchet.step()
    def step()
        ...
```